pkg/cloud-provider/cloudapi/azure/azure_query_helpers.go

```diff
@@ -161,7 +175,7 @@ func buildQueryForVpcIDOnlyMatches(vpcIDsWithVpcIDOnlyMatches map[string]struct{
 		return strings.Compare(vpcIDs[i], vpcIDs[j]) < 0
 	})
 
-	return getVMsByVnetIDsAndSubscriptionIDsAndTenantIDsAndLocationsMatchQuery(vpcIDs, subscriptionIDs, tenantIDs, locations)
```

Comment:
Missed changing the function name here
https://github.com/archanapholla/testrepo/blob/vnetMatch/pkg/cloud-provider/cloudapi/azure/azure_resourcegraph_vmTable.go#L160